packages/react-tinacms-editor/src/components/BaseMenubar/index.tsx

```diff
@@ -144,15 +121,13 @@ export const BaseMenubar = ({
       setMenuOffsetTop(getOffsetTop(wysiwygWrapper, stickyOffset))
     window.addEventListener('scroll', handleScrollStart)
     window.addEventListener('scroll', handleScrollStop)
-    window.addEventListener('resize', handleResize)
 
     return () => {
       window.removeEventListener('scroll', handleScrollStart)
       window.removeEventListener('scroll', handleScrollStop)
-      window.removeEventListener('resize', handleResize)
```

Comment:
Without this, the menubar width won't be recalculated on window resize when the menu is sticky

Reply:
Is that really an issue as long as the menu is floating as expected?

Reply:
@kendallstrautman what browser are you testing in? It's funny, this PR is intended to fix that exact problem!

Reply:
lol weird! I'm in firefox. I only see these weird states on window resize.

Reply:
right, I realized after posting the previous message that I was experiencing this sizing issue on initial page load. makes sense that removing the resize handler wouldn't recalculate it on resize

Reply:
I haven't been able to recreate the width sizing issue on page load in chrome that you've described....from what I just experienced in fixing the sticky behavior for chrome, it seems like chrome shows a greater lag than other browsers between `document.readyState === 'complete'` and the window's 'load' event. 

My guess is that maybe that initial `setMenuBoundingBox` call is happening before the content is actually loading. Could we try using `useLayoutEffect` instead or perhaps check if document.readyState is actually complete before we run that set state?